packages/app-staking/src/Account.tsx

```diff
@@ -114,6 +117,116 @@ class Account extends React.PureComponent<Props, State> {
     );
   }
 
+  private renderButtons () {
+    const { isInNominate } = this.state;
+
+    if (isInNominate) {
+      return null;
+    }
+
+    const { address, isIntending, nominating, t } = this.props;
+    const isNominating = !!nominating;
+    const canStake = !isIntending && !isNominating;
+
+    if (canStake) {
+      return (
+        <div className='staking--Account-buttons'>
+          <Button.Group>
+            <Button
+              isPrimary
+              onClick={this.stake}
+              text={t('account.stake', {
+                defaultValue: 'stake'
+              })}
+            />
+            <Button.Or />
+            <Button
+              isPrimary
+              onClick={this.toggleNominate}
+              text={t('account.nominate', {
+                defaultValue: 'nominate'
+              })}
+            />
+          </Button.Group>
+        </div>
+      );
+    }
+
+    if (isNominating) {
+      return (
+        <div className='staking--Account-buttons'>
+          <Button.Group>
+            <UnnominateButton
+              address={address || ''}
+              nominating={nominating || ''}
+              onClick={this.unnominate}
+            />
+          </Button.Group>
+        </div>
+      );
+    }
+
+    return (
+      <div className='staking--Account-buttons'>
+        <Button.Group>
+          <Button
+            isNegative
+            onClick={this.unstake}
+            text={t('account.unstake', {
+              defaultValue: 'unstake'
+            })}
+          />
+        </Button.Group>
+      </div>
+    );
+  }
+
+  private renderNominate () {
+    const { isInNominate, isNomineeValid, nominee } = this.state;
+
+    if (!isInNominate) {
+      return null;
+    }
+
+    const { t } = this.props;
+
+    return (
+      <div className='staking--Account-nominate'>
+        <div className='ui--row'>
+          <Input
```

Comment:
This really should be a validator dropdown. (However needs to be logged - we currently don't have a dropdown that takes an arbitrary list of accounts)